client/my-sites/checkout/cart/cart-total.jsx

```diff
@@ -31,6 +35,34 @@ class CartTotal extends React.Component {
 			return <div className="cart-total" />;
 		}
 
+		if ( this.isCouponApplied() ) {
+			return (
+				<>
+					<div className="cart-total">
+						<span className="cart-subtotal-label">{ translate( 'Base amount' ) }</span>
+						<span className="cart-subtotal-amount">{ cart.total_cost_before_coupon_display }</span>
+					</div>
+					<div className="cart-total">
+						<span className="cart-subtotal-label">
+							{ translate( 'Coupon applied "%(coupon)s"', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Coupon applied: %(coupon)s' )` *ES Score: 25*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).